contracts/ContractRegistry.sol

```diff
@@ -53,7 +77,7 @@ contract ContractRegistry is IContractRegistry, WithClaimableRegistryManagement
 		return managedContractAddresses;
 	}
 
-	function setManager(string calldata role, address manager) external onlyRegistryManager {
+	function setManager(string calldata role, address manager) external onlyManager {
```

Comment:
onlyManager is confusing with the managers. can use admin, perhaps call the registryAdmin and InitializationAdmin to separate them from the manager mechanism